pressbooks/pressbooks
# Change contributors list

Issue: https://github.com/pressbooks/pressbooks/issues/2231

This PR changes the Contributors list in Admin Book > Book Info > Contributors, and displays 4 columns:
- Picture
- Name (sortable)
- Institution (sortable)
- Biographical Info (sortable): Max 180 characteres

It also fixes: https://github.com/pressbooks/pressbooks/issues/2355

### Test
- Go to Admin Book > Book Info > Contributors and make sure the tables mentioned above are displayed in the order proposed.
- Make sure the sort actions works for all columns (except the Picture one, it should be disabled).

#### For https://github.com/pressbooks/pressbooks/issues/2355
- Edit a contributor with a profile picture.
- Change one or more fields, except for the picture one and save it.
- Make sure the contributor still has the profile picture in place.
